src/components/SearchPanel/index.css

```diff
@@ -1,6 +1,6 @@
 .search-panel-wrapper {
   min-height: 200px;
-  background-color: #C4C4C4;
+  background-color: #e5e5e5;
```

Comment:
![image](https://user-images.githubusercontent.com/22891941/75604634-cc529500-5aeb-11ea-976d-51f4be58c187.png)
I'm not sure that color aligned with design